packages/core-api/lib/versions/2/handlers/peers.js

```diff
@@ -18,7 +18,12 @@ exports.index = {
   async handler (request, h) {
     const allPeers = await blockchain.p2p.getPeers()
 
-    let result = allPeers.sort((a, b) => a.delay - b.delay)
+    let result = allPeers.map(peer => {
+        // just use 'OK' status for API instead of p2p http status codes
```

Comment:
For the v2 API we can simply return the numerical value as we don't need this OK hack for backwards compatibility.